Gopkg.toml

```diff
@@ -30,6 +30,10 @@
   name = "github.com/sergi/go-diff"
   version = "1.0.0"
 
+[[override]]
+  name = "github.com/coreos/prometheus-operator"
+  version = "v0.26.0"
```

Comment:
Should we start pinning all versions with `"=v..."`?

Reply:
Sounds like a good idea to me!

Reply:
Actually, this was not needed to be pinned in the end but I did pin it in the generated operator toml. 👍